diff --git a/base/BUILD.gn b/base/BUILD.gn
index d28dce0667b8c173064f591aeea435eca7dc46cf..f0c61e1a38d80f45b74cc370e91456ce97b8c61a 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4476,6 +4476,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
       "android/java/src/org/chromium/base/WrappedClassLoader.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -5006,6 +5007,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
