diff --git a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
index 2f14bc19a74d2bd0aa7c5c7a5dc6838c1bf5d00d..ad6a581d2cc1bf779ea9e8e5ae44c22bd3651362 100644
--- a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
+++ b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
@@ -750,6 +750,7 @@ SkPath IconLabelBubbleView::GetHighlightPath() const {
       SkIntToScalar(radii.lower_right()), SkIntToScalar(radii.lower_right()),
       SkIntToScalar(radii.lower_left()),  SkIntToScalar(radii.lower_left())};
 
+  BRAVE_ICON_LABEL_BUBBLE_VIEW_GET_HIGHLIGHT_PATH
   return SkPath().addRoundRect(rect, sk_radii);
 }
 
