diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 608c845713d133376ad266b1d51015cc1fd7635d..9d867cb1276e025a1a2748beaf194e9e05c7f887 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1440,6 +1440,7 @@ source_set("extensions") {
       allow_circular_includes_from += [ "//chrome/browser/glic:impl" ]
     }
   }  # if (enable_extensions)
+  deps += brave_extensions_resources
 }
 
 if (enable_extensions) {
