diff --git a/chrome/chrome_proxy/chrome_proxy_main_win.cc b/chrome/chrome_proxy/chrome_proxy_main_win.cc
index 4ba7702bfa283c5bc376dacbdcddd98c71b7ba04..5a4a462d89a93d47709b01e827bb33f66c11d16e 100644
--- a/chrome/chrome_proxy/chrome_proxy_main_win.cc
+++ b/chrome/chrome_proxy/chrome_proxy_main_win.cc
@@ -14,7 +14,10 @@
 namespace {
 
 constexpr base::FilePath::CharType kChromeExecutable[] =
+    FILE_PATH_LITERAL("kahf.exe");
+#if 0
     FILE_PATH_LITERAL("chrome.exe");
+#endif
 
 constexpr base::FilePath::CharType kChromeProxyExecutable[] =
     FILE_PATH_LITERAL("chrome_proxy.exe");
