diff --git a/chrome/android/javatests/BUILD.gn b/chrome/android/javatests/BUILD.gn
index abf859a09bba80e065e47d6bdf79e89630ccfff3..8943e595ff11589d23299efba21d69909da0ad89 100644
--- a/chrome/android/javatests/BUILD.gn
+++ b/chrome/android/javatests/BUILD.gn
@@ -1714,6 +1714,7 @@ android_library("chrome_unit_test_java_various") {
     "src/org/chromium/chrome/browser/toolbar/ToolbarSecurityIconTest.java",
     "src/org/chromium/chrome/browser/webapps/WebappLaunchCauseMetricsTest.java",
   ]
+  sources -= [ "src/org/chromium/chrome/browser/ntp/IncognitoDescriptionViewRenderTest.java" ]
 
   deps = chrome_unit_test_java_deps
 }
