diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index be679b3017a3eed007fad009dfa4d747e129845c..e88786a8797eb57afb5eefbd8af4c868f57f6914 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -488,6 +488,7 @@ content::WebUIDataSource* CreateAndAddExtensionsSource(Profile* profile,
           l10n_util::GetStringUTF16(IDS_SHORT_PRODUCT_OS_NAME)));
 #endif
 
+  BRAVE_CREATE_EXTENSIONS_SOURCE
   return source;
 }
 
