diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 494b731931bd7a56ffbcdbebb4b36766eff6bbfa..8bbbb761d88a96785c403b1918863fdb9e881d88 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -747,6 +747,7 @@ static_library("ui") {
 
     "//chrome/browser/task_manager:impl",
   ]
+  import("//brave/browser/sources.gni") public_deps += [ "//brave/browser/ui" ] deps += brave_chrome_browser_ui_allow_circular_includes_from allow_circular_includes_from += brave_chrome_browser_ui_allow_circular_includes_from
 
   if (enable_vr && is_win) {
     deps += [ "//chrome/browser/vr:vr_base" ]
@@ -3528,7 +3529,10 @@ static_library("ui") {
       ]
       deps += [ "//chrome/updater/app/server/win:updater_legacy_idl" ]
     } else {
-      sources += [ "webui/help/version_updater_basic.cc" ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [ "webui/help/version_updater_basic.cc" ]
+      }
     }
   }
 
@@ -5685,6 +5689,7 @@ static_library("ui_public_dependencies") {
     "//mojo/public/cpp/bindings",
     "//ui/webui/resources",
   ]
+  public_deps += [ "//brave/browser/ui:ui_public_dependencies" ]
   if (!is_android) {
     public_deps += [
       "//build:branding_buildflags",
