diff --git a/chrome/browser/resources/bookmarks/command_manager.ts b/chrome/browser/resources/bookmarks/command_manager.ts
index d935d00f5dba0d1a65bab2f25c0a24537811e395..426437d18001b141a7b0cbf9376b52d9fd687fb7 100644
--- a/chrome/browser/resources/bookmarks/command_manager.ts
+++ b/chrome/browser/resources/bookmarks/command_manager.ts
@@ -439,7 +439,7 @@ export class BookmarksCommandManagerElement extends
         chrome.bookmarkManagerPrivate.export();
         break;
       case Command.HELP_CENTER:
-        window.open('https://support.google.com/chrome/?p=bookmarks');
+        window.open('https://community.brave.com', undefined, 'noreferrer');
         break;
       default:
         assertNotReached();
