diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 4504187d037a49ea3d3751dc57b997b06207b646..1be5a1cf7a34612e95045223d7e5add5cd2fe670 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3620,6 +3620,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
