diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index f788b2526ad05563db450d2af82084cdb633bed7..bd9e35582999f3d31b4c4af7093567d24f19aadb 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3615,6 +3615,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
