diff --git a/ios/chrome/browser/shared/model/prefs/BUILD.gn b/ios/chrome/browser/shared/model/prefs/BUILD.gn
index d6306f4bd495a69f8d60a2b91a42f526d6aa7131..ce944e41a887f3af6d20b21e700a4c59ba653ad5 100644
--- a/ios/chrome/browser/shared/model/prefs/BUILD.gn
+++ b/ios/chrome/browser/shared/model/prefs/BUILD.gn
@@ -138,6 +138,7 @@ source_set("browser_prefs") {
     "//ios/web/common:features",
     "//ui/base",
   ]
+  import("//brave/chromium_src/ios/chrome/browser/shared/model/prefs/sources.gni") deps += brave_browser_prefs_deps
 }
 
 source_set("unit_tests") {
