diff --git a/ui/base/BUILD.gn b/ui/base/BUILD.gn
index 1d34a918374da3fa291c2fbcbbfca3a8ec0c89fc..a20b63852583c00751cee395dc3c16a7e489e8ff 100644
--- a/ui/base/BUILD.gn
+++ b/ui/base/BUILD.gn
@@ -548,6 +548,7 @@ component("base") {
       "//ui/events",
       "//ui/events/devices",
     ]
+    import("//brave/chromium_src/ui/base/sources.gni") deps += brave_chromium_src_ui_base_deps
     public_deps += [ "//ui/color:color_headers" ]
   }
 
