diff --git a/ui/webui/resources/cr_components/certificate_manager/certificate_manager_v2.ts b/ui/webui/resources/cr_components/certificate_manager/certificate_manager_v2.ts
index 1377ac5e0ede3ae30c171b105bf858d4a64b8557..2523331d9f9282cdc1c3926f78335a38218151e0 100644
--- a/ui/webui/resources/cr_components/certificate_manager/certificate_manager_v2.ts
+++ b/ui/webui/resources/cr_components/certificate_manager/certificate_manager_v2.ts
@@ -526,5 +526,5 @@ declare global {
   }
 }
 
-customElements.define(
-    CertificateManagerV2Element.is, CertificateManagerV2Element);
+// customElements.define(
+//     CertificateManagerV2Element.is, CertificateManagerV2Element);
