diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index ea4a470d2ba9592e5bd5eb875624aef50cfaf6c4..ae613319b05844565fe30e18bf7ebd79644fcc37 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3366,6 +3366,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
